correction: Added some elements

BinaryMindset opened the pull request.

BinaryMindset committed:
Added some elements

BinaryMindset commented:
added elements so more could be identified

sreekaransrinath commented:
Hey @gfunk81 , great job with adding the Geographical Information System files! However, each value in the key-value pair of the dictionary must be a path that files can be stored in. If you could fix that for the GIS filetypes, that'd be great!

Cheers ^^

BinaryMindset commented:
no clue not to edit and edit.

BinaryMindset committed:
put in proper section

incorrectly put GIS into own group. now broke down in the existing groups where they should of belong.

sreekaransrinath committed:
Update extensiondict.py
